train_network.py

```diff
@@ -0,0 +1,179 @@
+import tensorflow as tf
+import numpy as np
+from accuracy import *
+from DataReader import *
+from DRN import *
+import os
+from dice import *
+
+tf.reset_default_graph()
+directory = os.getcwd()
+
+train_file = os.path.join(directory, 'data', 'train.txt')
+test_file =  os.path.join(directory, 'data', 'val.txt')
+logs = os.path.join(directory, 'logs')
+trainloss = os.path.join(logs, 'train_loss.txt')
+
+
+if os.path.isdir(logs) == False:
+  os.makedirs(logs)
+
+# choose network, can be either DRN18 or DRN26
+network = 'DRN26'
+# set parameters
+batch_size=8
+num_epochs=100
+use_weights = 1
+num_classes = 5
+image_dims=[500,500,3]
+
+data = DataReader(directory, batch_size, num_epochs, use_weights=1)
```

Comment:
Why is `use_weights=1`? The variable name suggests that it might be better represented as a boolean (`use_weights=True`)